draft-ietf-lamps-rfc4210bis.md

```diff
@@ -1170,131 +1182,91 @@ other things) the certificate containing the public key of the
-  note that the subject and issuer fields must match);
 
-1. Verify that this is correct using the new CA key (which the
-  verifier has locally);
+1. Get the "new with new" and "new with old" certificate.
 
-1. If correct, check the signer's certificate using the old CA key.
 
-Case 2 will arise when the CA operator has issued the signer's
-certificate, then changed the key, and then issued the verifier's
-certificate; so it is quite a typical case.
 
+    1. If a repository is available, look up the caCertificate
+      attribute in the repository and pick the "new with new" and
+      "new with old" certificates.
 
-#### Verification in Case 3
-{: id="sect-4.4.2.3"}
+    1. If no repository is available, the verifier can use the root
+      CA update general message to request the new root CA and link
+      certificates from a PKI management entity, see {{sect-5.3.19.15}}.
 
-In case 3, the verifier must get access to the new public key of the
-CA.  In case a repository is used, the verifier does the following:
+    1. Otherwise, get it "out-of-band" in a trustworthy manner.
 
 
-1. Look up the cACertificate attribute in the repository and pick
-  the NewWithOld certificate (determined based on validity periods;
-  note that the subject and issuer fields must match);
-
-1. Verify that this is correct using the old CA key (which the
-  verifier has stored locally);
+1. If received the two certificates, check that the validity periods
+  and the subject and issuer fields match. Verify the signatures
+  using the old CA key (which the verifier has locally). Securely
+  store the new trust anchor information if all checks were
+  successful.
 
 1. If correct, check the signer's certificate using the new CA key.
 
-Case 3 will arise when the CA operator has issued the verifier's
-certificate, then changed the key, and then issued the signer's
-certificate; so it is also quite a typical case.
 
-Note: Alternatively, the verifier can use the root CA update general message
-to request the respective certificates from a PKI management entity, see {{sect-5.3.19.15}}, and follow the required validation steps.
+#### Verification in Case 3
+{: id="sect-4.4.2.3"}
+
+In case 3, the verifier must get access to the old public key of the
+CA. Case 3 will arise when the CA operator has issued the signer's
+certificate, then changed the key, and then issued the verifier's
+certificate.
 
+The verifier does the following:
 
-#### Failure of Verification in Case 6
-{: id="sect-4.4.2.4"}
 
-In this case, the CA has issued the verifier's PSE, which contains
-the new key, without updating the repository attributes.  This means
-that the verifier has no means to get a trustworthy version of the
-CA's old key and so verification fails.
+1. Get the "old with new" certificate.
 
-Note that the failure is the CA operator's fault.
 
 
-#### Failure of Verification in Case 7
-{: id="sect-4.4.2.5"}
+    1. If a repository is available, look up the caCertificate
```

Comment:
Same comment as before, the AIA extension id-ad-caIssuers could be used to get the certificates (From an LDAP, or HTTP or somewhere else).   We usually treat the two operations as separate things to be checked.  For example, we would use the AIA if it exists, and also lookup the caCertificate attribute.   If no successful path can be built to the desired target (either the old or new CA) then we fail because we didn't have access to the link certificates.